doc/rst/lib_spi.rst

```diff
@@ -823,10 +939,360 @@ As application runs that reads a value from the SPI connected WiFi chip and prin
+been downloaded and installed (see `README` for required version).
+
+Installation instructions can be found `here <https://xmos.com/xtc-install-guide>`_. Particular
+attention should be paid to the section `Installation of required third-party tools
+<https://www.xmos.com/documentation/XM-014363-PC-10/html/installation/install-configure/install-tools/install_prerequisites.html>`_.
+
+The application uses the `XMOS` build and dependency system, `xcommon-cmake <https://www.xmos.com/file/xcommon-cmake-documentation/?version=latest>`_. `xcommon-cmake`
+is bundled with the `XMOS` XTC tools. It runs on the `xcore.ai` evaluation kit, `XK-EVK-XU316 <https://www.xmos.com/xk-evk-xu316>`_.
+
+To configure the build, run the following from an XTC command prompt:
+
+.. code-block:: console
+
+  cd examples
+  cd app_spi_slave
+  cmake -G "Unix Makefiles" -B build
+
+Any missing dependencies will be downloaded by the build system at this configure step.
+
+
+Finally, the application binaries can be built using ``xmake``:
+
+.. code-block:: console
+
+  xmake -j -C build
+
+
+Running
+.......
+
+To run the application return to the ``/examples/app_spi_slave`` directory and run the following command:
+
+.. code-block:: console
+
+  xsim --xscope '-offline trace.xmt' bin/app_spi_slave.xe  \
+  --trace-plugin VcdPlugin.dll '-tile tile[0] -o trace.vcd -xe bin/app_spi_slave.xe \
+  -ports -functions -cores -instructions' --plugin LoopbackPort.dll \
+  '-port tile[0] XS1_PORT_1I 1 0 -port tile[0] XS1_PORT_1E 1 0 \
+  -port tile[0] XS1_PORT_1J 1 0 -port tile[0] XS1_PORT_1F 1 0 \
+  -port tile[0] XS1_PORT_1K 1 0 -port tile[0] XS1_PORT_1G 1 0 \
+  -port tile[0] XS1_PORT_1L 1 0 -port tile[0] XS1_PORT_1H 1 0' 
+
+.. note::
+
+    This command line is provided as a file in the ``/examples/app_spi_slave`` directory under the filename ``simulate_cmd.txt``.
+    You can rename this file to ``simulate_cmd.sh`` or ``simulate_cmd.bat`` and run it directly, depending on your host OS.
+
+As application runs that reads a value from the SPI connected WiFi chip and prints the following output to the console::
+
+  APP: Set register 0 to 0xED
+  APP: Register 0 is 0xED, Register 1 is 0x0
+  APP: Register 0 is 0xED, Register 1 is 0x0
+  SPI MASTER: Read register 0: 0xED
+  APP: Register 0 is 0xED, Register 1 is 0x0
+  SPI MASTER: Set register 1 to 0xAC
+  APP: Register 0 is 0xED, Register 1 is 0xAC
+  APP: Register 0 is 0xED, Register 1 is 0xAC
+  APP: Register 0 is 0xED, Register 1 is 0xAC
+
+Both registers were initialised to 0x00 so you can see the successful application side write to register 0 of value 0xED, followed by the
+SPI master read of that register shortly afterwards. You can also see that the SPI master writes to register 1 with the value of 0xAC
+which is then successfully read by the application.
+
+Next, you can also view the simulation in a VCD (Voltage Change Description) viewer such as ``gtkwave`` by running the following command:
```

Comment:
I think tis is more than a suggestion to use gtkwave if the instructions point to the .gtkw file. I would be more firm with the language i.e. "using gtkwave" or leave it as a suggestion and point to the .vcd file.